.pre-commit-config.yaml

```diff
@@ -2,4 +2,27 @@ repos:
 -   repo: https://github.com/psf/black
     rev: 19.10b0
     hooks:
-    - id: black
\ No newline at end of file
+    - id: black
+-   repo: https://github.com/asottile/seed-isort-config
+    rev: v2.1.1
+    hooks:
+    -   id: seed-isort-config
+-   repo: https://github.com/timothycrosley/isort
+    rev: 4.3.21
+    hooks:
+    -   id: isort
```

Comment:
`isort` is now in v5 and includes a pre-commit hook that doesn't need `seed-isort-config` anymore. See this guide https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0/#migrating-pre-commit to upgrade

Reply:
Also it now supports black formatting natively, you just need to configure it. See the docs there https://pycqa.github.io/isort/docs/configuration/compatibility_black/ (I think you will need to put it both in the configuration file and in the pre-commit `args`)